llvm/test/CodeGen/AArch64/ptrauth-indirectbr.ll

```diff
@@ -0,0 +1,106 @@
+; RUN: llc -mtriple arm64e-apple-darwin \
```

Comment:
does update_test_checks do horrible things to the CHECK lines in this?

Reply:
It's not too bad; switched